packages/vendure-plugin-shipping-extensions/test/shipping-extensions.spec.ts

```diff
@@ -0,0 +1,409 @@
+import {
+  createTestEnvironment,
+  registerInitializer,
+  SimpleGraphQLClient,
+  SqljsInitializer,
+  testConfig,
+} from '@vendure/testing';
+import { initialData } from '../../test/src/initial-data';
+import {
+  DefaultLogger,
+  LogLevel,
+  mergeConfig,
+  ProductService,
+  ProductVariantService,
+  defaultShippingEligibilityChecker,
+  RequestContext,
+  roundMoney,
+} from '@vendure/core';
+import { TestServer } from '@vendure/testing/lib/test-server';
+import { testPaymentMethod } from '../../test/src/test-payment-method';
+import { getSuperadminContext } from '@vendure/testing/lib/utils/get-superadmin-context';
+import { createSettledOrder } from '../../test/src/shop-utils';
+import { ShippingExtensionsPlugin } from '../src/shipping-extensions.plugin';
+import gql from 'graphql-tag';
+import { expect, describe, beforeAll, afterAll, it, vi, test } from 'vitest';
+import { distanceBasedShippingCalculator } from '../src/distance-based-shipping-calculator';
+import {
+  POSTCODES_URL,
+  UKPostalCodeToGelocationConversionStrategy,
+} from '../src/strategies/uk-postalcode-to-geolocation-strategy';
+import { GeoLocation } from '../src/strategies/order-address-to-geolocation-strategy';
+import nock from 'nock';
+import { getDistanceBetweenPointsInKMs } from '../src/get-distance-between-points';
+import { CreateAddressInput } from '../../test/src/generated/shop-graphql';
+
+describe('Order export plugin', function () {
```

Comment:
You can remove the outter `describe` (this one even has a typo in it, probably my typo :laughing: )